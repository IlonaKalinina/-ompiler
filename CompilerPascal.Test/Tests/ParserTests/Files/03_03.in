program Test_03_03;
var
i : integer;
begin
if i > 5 then write('yes') else write('no');
end.