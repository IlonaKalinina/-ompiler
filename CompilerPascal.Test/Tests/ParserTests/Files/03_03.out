program Test_03_03
├───var
│    └─── :
│         ├─── i
│         └─── integer
│    
begin
├─── if
│    ├─── >
│    │    ├─── i
│    │    └─── 5
│    ├─── write
│    │    └─── yes
│    else
│    └─── write
│         └─── no
└─── end
